streampipes-extensions/streampipes-connectors-opcua/src/main/java/org/apache/streampipes/extensions/connectors/opcua/adapter/configuration/SpOpcUaConfigBuilder.java

```diff
@@ -102,26 +94,15 @@ public static SpOpcUaConfig from(IStaticPropertyExtractor extractor) {
             extractor.singleValueParameter(OPC_SERVER_URL.name(), String.class);
         serverAddress = OpcUaUtil.formatServerAddress(serverAddress);
 
-        return new SpOpcUaConfig(serverAddress,
-            namespaceIndex,
-            nodeId,
-            username,
-            password,
-            pullIntervalSeconds,
+        return new SpOpcUaConfig(serverAddress, 1, null, username, password, pullIntervalSeconds,
```

Comment:
Maybe you can change the formatting back. This makes it easier to review on small devices or was there a checkstyle problem?
Can the nodeId & namespaceIndex be removed from the **SpOocUaConfig**?

Reply:
Yes, we should remove this later as part of the adapter refactoring, I just wanted to focus on the improved browsing for this PR to avoid too many changes.

Reply:
I added this to https://github.com/apache/streampipes/discussions/1650